### PR TITLE
Remove signatures that duplicate introspectable data.

### PR DESCRIPTION
**Pull request by anntzer**
(For the case of Axes.table, one can just directly use the
`matplotlib.table.table` function as the method.)

## PR Checklist

- [ ] Has Pytest style unit tests
- [ ] Code is [Flake 8](http://flake8.pycqa.org/en/latest/) compliant
- [ ] New features are documented, with examples if plot related
- [ ] Documentation is sphinx and numpydoc compliant
- [ ] Added an entry to doc/users/next_whats_new/ if major new feature (follow instructions in README.rst there)
- [ ] Documented in doc/api/api_changes.rst if API changed in a backward-incompatible way

### CODE REVIEW

File: lib/matplotlib/axes/_axes.py [hunk 1]
```diff
@@ -6986,12 +6956,6 @@ def psd(self, x, NFFT=None, Fs=None, Fc=None, detrend=None,
         r"""
         Plot the power spectral density.
 
-        Call signature::
-
-          psd(x, NFFT=256, Fs=2, Fc=0, detrend=mlab.detrend_none,
```
Comment (QuLogic) [inline lib/matplotlib/axes/_axes.py]: The default `detrend` here says it's 'none', but the new docstring is a bit vague about it, and seems like it should be 'default' (which is 'mean'). I think the actual default _is_ 'none' though.

File: lib/mpl_toolkits/mplot3d/axes3d.py [hunk 2]
```diff
@@ -2735,19 +2735,14 @@ def calc_arrow(uvw, angle=15):
 
     def voxels(self, *args, facecolors=None, edgecolors=None, **kwargs):
         """
-        ax.voxels([x, y, z,] /, filled, **kwargs)
+        ax.voxels([x, y, z,] /, filled, facecolors=fc, edgecolors=ec, **kwargs)
```
Comment (QuLogic) [inline lib/mpl_toolkits/mplot3d/axes3d.py]: Should the kwargs be `None` instead?

Comment (anntzer) [reply lib/mpl_toolkits/mplot3d/axes3d.py]: sorry, had just copy-pasted from below, now fixed.

File: lib/matplotlib/axes/_axes.py (hunk 1 above)
Comment (anntzer) [reply lib/matplotlib/axes/_axes.py]: Thanks for catching that.
I edited the PSD signature in mlab.py to not list "default" and "constant" anymore (and explicitly indicate that "none" is the default), with the plan to deprecate "default" and "constant" in a separate PR (having a default that's not "default" is just... confusing).

File: lib/matplotlib/mlab.py [hunk 3]
```diff
@@ -669,15 +669,15 @@ def _single_spectrum_helper(x, mode, Fs=None, window=None, pad_to=None,
         This should *NOT* be used to get zero padding, or the scaling of the
         result will be incorrect. Use *pad_to* for this instead.
 
-    detrend : {'default', 'constant', 'mean', 'linear', 'none'} or callable
+    detrend : {'none', 'mean', 'linear'} or callable, default 'none'
```
Comment (anntzer) [inline lib/matplotlib/mlab.py]: see https://github.com/matplotlib/matplotlib/pull/13177#discussion_r248549789.

**Pull request merged**